x-pack/plugins/cases/server/saved_object_types/migrations/user_actions/types.ts

```diff
@@ -15,7 +15,7 @@ export interface UserActions {
   owner: string;
 }
 
-export interface UserActionUnmigratedConnectorDocument {
+export interface UserActionVersion800 {
```

Comment:
I'm happy to name this something else, the reason I changed it is because it's being used for the new `removeRuleInformation` migration and a few other ones.

Reply:
We can give a name that distinguishes the old payload (new_value, old_value) from the new one (`payload: { connector, ... }`)